### ISSUE TITLE
How to cope with shared views between back and frontend

### ISSUE THREAD

**Issue by aad4** [joomleague/JoomLeague#74]
Currently when a user logs into the frontend and visits a Joomleague page that has editing capability (ClubInfo, Player, Referee, Results, Roster, Staff, Teaminfo), and then clicks on the "Edit" icon, JL will automatically try to log the user into the backend, to show the backend view for editing of this "item".

The idea behind it was to minimize the maintenance of this functionality, by not having to code these parts twice (once for backend and once for frontend). It also prevents that the implementation for front and backend deviate from each other (maybe even different functionality, which could lead to user confusion).

Some issues:
* the logging in does not always go smooth
* the frontend users who are appointed to fill in e.g. the scores of his/her team every week needs to have access to the backend (ACL-wise).

I think latter issue is not a desirable situation.

Can we come up with a different approach? E.g. create a shared layout for front- and backend? What about the involved controllers and models? Could those be shared as well?

**Comment by aad4** [comment]
Some interesting links on this topic:
https://docs.joomla.org/J3.x:Sharing_layouts_across_views_or_extensions_with_JLayout
https://docs.joomla.org/J3.x:JLayout_Improvements_for_Joomla!
https://github.com/joomla-projects/cms-naked/issues/2
http://stackoverflow.com/questions/14747072/how-to-show-the-backend-list-in-joomla-frontend

@Jaap & @Heart1010 :
- do you guys have experience with it?
- have you seen it being used for forms (so not just displaying of data, but also entry of data)?
- could this be a simplification of the "extension mechanism" that is currently present (you can override layouts by placing your own rendering version in the proper path)? The extension mechanism can do much more, but do we really need it? I like KISS approach.

**Comment by JKoelman** [comment]
> do you guys have experience with it?
Not really, did knew about JLayout and did experiment a bit with it. The thing was that i wanted to use the current styling of Joomla 3 for using search/filters in the backend so had to do some trial and error to test things out and sorted that part out. But for the rest i don't know.

**Comment by Heart1010** [comment]
> do you guys have experience with it? 

Sorry, no... can't be of any help in this topic.
